data/competencies.yml

```diff
@@ -1188,3 +1188,17 @@
   level: senior2
   area: leadership
   domain: null
+
+# S2
+- id:
+  summary: Actively encourages or shapes the personal development of more junior people
+  examples:
+    - Helps other engineers to form or meet specific career goals
+    - Organises conference attendance for a group of engineers
+    - Considerately gives targeted feedback with examples of a behaviour and its impact
+    - TODO Something about sponsorship
```

Comment:
I can't find the right wording for this one. This is the idea I'd like to capture is something like: 
```
Finds or creates opportunities for others to develop new skills or build their confidence
```